src/app/components/options-sidebar/time-clock/time-clock.component.ts

```diff
@@ -101,10 +108,24 @@ export class TimeClockComponent  implements OnInit {
   }
 
   setTimeToInOut() {
```

Comment:
what is this function doing?
its name is weird :(

Reply:
function to establish arrival and departure times for employees